maps/away/ships/elyra/elyra_corvette/elyra_corvette.dm

```diff
@@ -0,0 +1,171 @@
+	suffixes = list("ships/elyra/elyra_corvette/elyra_corvette.dmm")
+	sectors = list(SECTOR_VALLEY_HALE, SECTOR_BADLANDS, SECTOR_NEW_ANKARA, SECTOR_AEMAQ)
+	spawn_weight = 1
+	ship_cost = 1
+	id = "elyran_corvette"
+	shuttles_to_initialise = list(/datum/shuttle/autodock/overmap/elyran_shuttle)
+
+	unit_test_groups = list(3)
+
+/singleton/submap_archetype/elyran_corvette
+	map = "Elyran Corvette"
+	descriptor = "One of the first vessels from Elyra's recent military modernization efforts to enter active service, the Sahin-class has taken great strides in improved quality and survivability from previous designs and is on track to become the backbone of the Elyran Republic's border control efforts. Equipped and crewed to handle anti-piracy operations, border patrols, and even to assist with disaster relief, this vessel follows the Elyran Armed Force's doctrine of versatility and is capable of striking out on its own for weeks at a time without resupply if required."
+
+/obj/effect/overmap/visitable/ship/elyran_corvette
+	name = "Elyran Corvette"
+	class = "ENV"
+	desc = "One of the first vessels from Elyra's recent military modernization efforts to enter active service, the Sahin-class has taken great strides in improved quality and survivability from previous designs and is on track to become the backbone of the Elyran Republic's border control efforts. Equipped and crewed to handle anti-piracy operations, border patrols, and even to assist with disaster relief, this vessel follows the Elyran Armed Force's doctrine of versatility and is capable of striking out on its own for weeks at a time without resupply if required."
+	icon_state = "corvette"
+	moving_state = "corvette_moving"
+	colors = list("#ffae17", "#ffcd70")
+	scanimage = "elyran_corvette.png"
+	designer = "Jewel Aerospace, Republic of Elyra"
+	volume = "52 meters length, 44 meters beam/width, 18 meters vertical height"
+	drive = "Low-Speed Warp Acceleration FTL Drive"
+	weapons = "Dual extruding fore-mounted medium caliber ballistic armament, fore obscured flight craft bay"
+	sizeclass = "Sahin-class Corvette"
+	shiptype = "Military patrol and combat utility"
+	max_speed = 1/(2 SECONDS)
+	burn_delay = 1 SECONDS
+	vessel_mass = 5000
+	fore_dir = SOUTH
+	vessel_size = SHIP_SIZE_SMALL
+	initial_restricted_waypoints = list(
+		"Elyran Naval Shuttle" = list("nav_hangar_elyra")
+	)
+
+	initial_generic_waypoints = list(
+		"nav_elyran_corvette_1",
+		"nav_elyran_corvette_2",
+		"nav_elyran_corvette_3",
+		"nav_elyran_corvette_4",
+		"nav_elyran_corvette_dock_port",
+		"nav_elyran_corvette_dock_starboard"
+	)
+
+	invisible_until_ghostrole_spawn = TRUE
+
+/obj/effect/overmap/visitable/ship/elyran_corvette/New()
+	designation = "[pick("Persepolis", "Damascus", "Medina", "Aemaq", "New Suez", "Bursa", "Republican", "Falcon", "Gelin", "Sphinx", "Takam", "Dandan", "Anqa", "Falak", "Uthra", "Djinn", "Roc", "Shadhavar", "Karkadann")]"
```

Comment:
A corvette shouldn't have the name of a planet, that's usually something reserved for capital ships. Pick less-important cities instead

Reply:
These names were copied over from the existing elyran strike craft.

Reply:
Change those too then, please